06_create_cluster.sh

```diff
@@ -12,6 +12,12 @@ source validation.sh
 
 early_deploy_validation
 
+if [[ ! -z "$INSTALLER_PROXY" && "$IP_STACK" = "v4" ]]; then
```

Comment:
My question is just curiosity and to learn;

Why we only support IPv4?

Reply:
Just for the sake of simplicity right now, but the long-term goal is to cover ipv6 as well